packages/pyml/pyml.20171117/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "Thierry Martinez <martinez@nsup.org>"
+authors: "Thierry Martinez <martinez@nsup.org>"
+homepage: "http://pyml.gforge.inria.fr"
+bug-reports: "http://pyml.gforge.inria.fr/tracker"
+license: "BSD"
+dev-repo: "https://scm.gforge.inria.fr/anonscm/git/pyml/pyml.git"
+build: [make "all" "pymltop" "pymlutop" {utop:installed} "PREFIX=%{prefix}%"]
+install: [make "install" "PREFIX=%{prefix}%"]
+remove: [make "uninstall" "PREFIX=%{prefix}%"]
+depends: [
+  "ocamlfind" {build}
+  "stdcompat" {>= "1"}
+]
+depopts: ["utop"]
+version: "20171117"
```

Comment:
Minimum version please :) `3.12` as you said.

Reply:
Fixed! Thanks @rgrinberg .